test/e2e/helpers.js

```diff
@@ -101,21 +110,40 @@ async function withFixtures(options, testSuite) {
   let driver;
   let failed = false;
   try {
-    if (!disableGanache) {
+    if (ganacheServer) {
       await ganacheServer.start(ganacheOptions);
     }
+
+    if (localNetworkServer) {
+      await localNetworkServer.start(networkOptions || {});
+    }
+
     let contractRegistry;
 
-    if (smartContract && !disableGanache) {
-      const ganacheSeeder = new GanacheSeeder(ganacheServer.getProvider());
-      const contracts =
-        smartContract instanceof Array ? smartContract : [smartContract];
-      await Promise.all(
-        contracts.map((contract) =>
-          ganacheSeeder.deploySmartContract(contract),
-        ),
-      );
-      contractRegistry = ganacheSeeder.getContractRegistry();
+    if (smartContract) {
+      if (ganacheServer) {
```

Comment:
once we have migrated all tests, we'll be able to remove this code block and use the below one with the AnvilSeeder